modules/client/src/internal/utils.ts

```diff
@@ -37,43 +34,43 @@ import { Result } from "@ethersproject/abi";
 import { keccak256 } from "@ethersproject/keccak256";
 import { toUtf8Bytes } from "@ethersproject/strings";
 
-export function unwrapCreateDaoParams(
-  params: ICreateParams,
-  metadataCid: string,
-): [DAOFactory.DAOConfigStruct, DAOFactory.VoteConfigStruct, string, string] {
-  // TODO: Serialize plugin params into a buffer
-  const pluginDataBytes = "0x" +
-    params.plugins
-      .map((entry) => {
-        const item = pack(["uint256", "bytes[]"], [entry.id, entry.data]);
-        return strip0x(item);
-      })
-      .join("");
+// export function unwrapCreateDaoParams(
+//   params: ICreateParams,
+//   metadataCid: string,
+// ): [DAOFactory.DAOConfigStruct, DAOFactory.VoteConfigStruct, string, string] {
+//   // TODO: Serialize plugin params into a buffer
+//   const pluginDataBytes = "0x" +
+//     params.plugins
+//       .map((entry) => {
```

Comment:
If this wrapper did the work for the legacy contracts, let's either remove or use+adapt this. But not leaving dead comments here, please